firstvoices/backend/urls.py

```diff
@@ -53,6 +54,9 @@
 sites_router.register(r"images", ImageViewSet, basename="image")
 sites_router.register(r"people", PersonViewSet, basename="person")
 sites_router.register(r"search", SiteSearchViewsSet, basename="site-search")
+sites_router.register(
+    r"dictionary-search", DictionarySearchViewSet, basename="dictionary-search"
```

Comment:
Any reason not to just have this be the list view of /dictionary like it was for the old api?

Reply:
+1 to this question

Reply:
+1 for /dictionary/search as well, but yes it's totally just stylistic